macOSUpgrade.sh

```diff
@@ -444,29 +456,38 @@ else
     jamfHelperPID=$!
 fi
 
-##Load LaunchAgent
+## Load LaunchAgent
 if [ "$fvStatus" = "FileVault is On." ] && \
    [ "$currentUser" != "root" ] && \
    [ "$cancelFVAuthReboot" -eq 0 ] ; then
     userID=$( /usr/bin/id -u "${currentUser}" )
     /bin/launchctl bootstrap gui/"${userID}" /Library/LaunchAgents/com.apple.install.osinstallersetupd.plist
 fi
 
-##Begin Upgrade
-/bin/echo "Launching startosinstall..."
```

Comment:
Change using https://github.com/kc9wwh/macOSUpgrade/pull/121/files#diff-a010e0d389262cc881767e742ac2c4e0R488 instead of this line.